tests/integration/security/rbac/v2/basic/basic_test.go

```diff
@@ -80,72 +54,75 @@ func TestRBACV2(t *testing.T) {
+			ExpectedRespCode: connection.DenyHTTPRespCode},
 		{Request: connection.Connection{To: appC, From: appB, Port: 80, Protocol: apps.AppProtocolHTTP, Path: "/credentials/admin"},
-			ExpectAllowed: false, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.DenyHTTPRespCode},
 		{Request: connection.Connection{To: appC, From: appB, Port: 90, Protocol: apps.AppProtocolHTTP},
-			ExpectAllowed: false, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.DenyHTTPRespCode},
 		{Request: connection.Connection{To: appC, From: appD, Port: 80, Protocol: apps.AppProtocolHTTP, Path: "/"},
-			ExpectAllowed: isMtlsEnabled, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: respCodeFromMtls[isMtlsEnabled]},
 		{Request: connection.Connection{To: appC, From: appD, Port: 80, Protocol: apps.AppProtocolHTTP, Path: "/any_path/admin"},
-			ExpectAllowed: false, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.DenyHTTPRespCode},
 		{Request: connection.Connection{To: appC, From: appD, Port: 90, Protocol: apps.AppProtocolHTTP},
-			ExpectAllowed: false, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.DenyHTTPRespCode},
 
 		{Request: connection.Connection{To: appD, From: appA, Port: 80, Protocol: apps.AppProtocolHTTP, Path: "/xyz"},
-			ExpectAllowed: true, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.AllowHTTPRespCode},
 		{Request: connection.Connection{To: appD, From: appA, Port: 90, Protocol: apps.AppProtocolHTTP},
-			ExpectAllowed: false, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.DenyHTTPRespCode},
 		{Request: connection.Connection{To: appD, From: appB, Port: 80, Protocol: apps.AppProtocolHTTP, Path: "/"},
-			ExpectAllowed: true, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.AllowHTTPRespCode},
 		{Request: connection.Connection{To: appD, From: appB, Port: 90, Protocol: apps.AppProtocolHTTP},
-			ExpectAllowed: false, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.DenyHTTPRespCode},
 		{Request: connection.Connection{To: appD, From: appC, Port: 80, Protocol: apps.AppProtocolHTTP, Path: "/any_path"},
-			ExpectAllowed: true, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.AllowHTTPRespCode},
 		{Request: connection.Connection{To: appD, From: appC, Port: 90, Protocol: apps.AppProtocolHTTP},
-			ExpectAllowed: false, RejectionCode: connection.DenyHTTPRespCode},
+			ExpectedRespCode: connection.DenyHTTPRespCode},
 	}
 
 	testDir := ctx.WorkDir()
-	testNameSpace := appInst.Namespace().Name()
-	rbacTmplFiles := []string{rbacClusterConfigTmpl, rbacV2RulesTmpl}
-	rbacYamlFiles := util.GetRbacYamlFiles(t, testDir, testNameSpace, rbacTmplFiles)
-
-	policy.ApplyPolicyFiles(t, env, testNameSpace, rbacYamlFiles)
+	testNamespace := appInst.Namespace().Name()
+	fmt.Println("Test namespace")
```

Comment:
Consider removing these two Println() statements.

Reply:
Forgot to remove them. Done. Thanks Lei.